extensions/amp-consent/0.1/consent-info.js

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+import {
+  CONSENT_STRING_TYPE, // eslint-disable-line no-unused-vars
```

Comment:
nit: You should be able to get rid of this by using `{../../../src/consent-state.CONSENT_STRING_TYPE}`

Reply:
hmm I don't see this anywhere in the codebase and I get an error when I do this... Anywhere I can see an example of this?

Reply:
For example https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/amp-a4a.js#L271

Reply:
please remove the eslint comment. we no longer need it